config/initializers/00_version.rb

```diff
@@ -0,0 +1,3 @@
+# Log versions
+Rails.logger.info "Starting Keystorm/#{Keystorm::VERSION}" \
```

Comment:
If you start the server in `apache2` instead of `puma`, this will likely fail ... because of the `::Puma::Const::VERSION` reference. If you would like to keep it, try `defined?(::Puma) ? ::Puma::Const::VERSION : 'no-puma'`.